frontend/rust-lib/flowy-grid/src/services/field/type_options/selection_type_option/select_option.rs

```diff
@@ -322,15 +324,24 @@ impl SelectOptionCellChangeset {
     }
 }
 
+
+/// [SelectOptionCellDataPB] contains a list of user's selected options and a list of all the options
+/// that the cell can use.
 #[derive(Clone, Debug, Default, Serialize, Deserialize, ProtoBuf)]
 pub struct SelectOptionCellDataPB {
+
+    /// The available options that the cell can use.
```

Comment:
I think that docs for inside the function should be //! , not ///
I'm still trying to figure this out